### ISSUE TITLE
Out of date tuning documentation

### ISSUE THREAD

**Issue by kahmed10** [ROCm/Tensile#1310]
Recent updates to the scripts (#1258?) under Tensile/tuning/scripts are not reflected under Tensile/tuning/README.md. For example, `provision_tuning.sh` has a new format:
```
Usage: ./tuning/scripts/provision_tuning.sh WORKING_PATH LOG_PATH OUTPUT_SUFFIX.yaml LIBRARY [options]
```
But the documentation is showing an example with the following args:
```
./tuning/scripts/provision_tuning.sh -w tensile_tuning -z logs/inception_rocblas-configs_unique.log -r tensile_tuning/tensile/Tensilen-rocblas-configs_unique.log -o tf_inception.yaml -y sgemm -l vega20
```

**Comment by benjaminulmer** [comment]
Good catch. Looks like this README hasn't been updated in quite some time. 

In the meantime, the [tuning doc](https://github.com/ROCmSoftwarePlatform/Tensile/tree/develop/tuning_docs) has more up-to-date information on how to use these scripts.

**Comment by kahmed10** [comment]
Thanks for the update. Please close this issue when the documentation gets updated.

**Comment by ppanchad-amd** [comment]
@kahmed10 Internal ticket has been created to fix documentation. Thanks!

**Comment by ppanchad-amd** [comment]
README file is now updated. Closing ticket.
